source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,268 @@
+
+Creating checklists
+~~~~~~~~~~~~~~~~~~~
+
+Checklists are created within a Playbook and provide steps to follow during an incident. Checklists can be created ahead of time as part of a Playbook, or during an incident.
+
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can add steps and edit checklist items during an active incident. You can also edit checklist items in Playbooks that are not being used in active incidents. 
+
+To edit a checklist, choose **Edit**.
+
+- Select the text field to edit the item details. 
+- Hover over an input box and select the **X** to remove the checklist item.
+- Drag the hamburger menu to rearrange the checklist items.
+
+Choose **Done** to save the changes.
+
+
+Ending incidents
+~~~~~~~~~~~~~~~~
+
+There are two ways to end an incident:
+
+- Using slash command */incident end* from within the incident channel.
+- Using the **End Incident** button in the RHS panel while in the incident channel.
+
+The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+
+Administrator's Guide
+^^^^^^^^^^^^^^^^^^^^^^
+
+Permissions
+~~~~~~~~~~~~~~~~~~~~~
+
+Incidents and Playbooks are associated with teams in Mattermost, and permissions are assigned based on the Permission scheme used in Mattermost. Incidents are directly tied to a specific team and channel.
+
+Public incidents
+~~~~~~~~~~~~~~~~~~~~~
+
+Public incidents are incidents where anyone with access to the team the incident channel is created on can join the incident as a participant by joining the incident channel. Like public channels, public incidents appear for everyone in the team they are created in. Any member of the team can join the primary channel and become an incident participant. 
+
+Private incidents
+~~~~~~~~~~~~~~~~~~~~~
+
+Private incidents are invite-only. Participants can be added by inviting them to the incident's primary channel. Like private channels, private incidents do not appear on the incident list of users who are not participants.
+
+To view incidents associated with your Mattermost team, select **! Incidents** from the plugin menu in the channel header to open the RHS menu. If there are no active incidents, you can start one or you can view incident logs.
+
+Incidents Backstage
+~~~~~~~~~~~~~~~~~~~~
+
+To open the Incidents backstage, which lists all incidents associated with the current team, select **Incidents & Playbooks** from the Main Menu. All incidents for the current team are listed for review with the following details for each incident:
+
+- Name
+- Status (**Ongoing** or **Ended**)
+- Start Date
+- End Date, if ended, otherwise **--**
+- Commander
+
+All dates are rendered as per the location and timezone of the active user.
```

Comment:
nit: This /seems/ like the kind of detail that belongs in the design doc, but is irrelevant for an enduser. (Unless it /didn't/ render in the expected timezone.)